monai/losses/unified_focal_loss.py

```diff
@@ -0,0 +1,233 @@
+# Copyright (c) MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import warnings
+from typing import Union
+
+import torch
+from torch.nn.modules.loss import _Loss
+
+from monai.networks import one_hot
+from monai.utils import LossReduction
+
+
+class AsymmetricFocalTverskyLoss(_Loss):
+    """
+    AsymmetricFocalTverskyLoss is a variant of FocalTverskyLoss, which attentions to the foreground class.
+
+    Reimplementation of the Asymmetric Focal Tversky Loss (with a build-in sigmoid activation) described in:
+
+    - "Unified Focal Loss: Generalising Dice and Cross Entropy-based Losses to Handle Class Imbalanced Medical Image Segmentation",
+    Michael Yeung, Computerized Medical Imaging and Graphics
+    """
+
+    def __init__(
+        self,
+        to_onehot_y: bool = False,
+        delta: float = 0.7,
+        gamma: float = 0.75,
+        epsilon: float = 1e-7,
+        reduction: Union[LossReduction, str] = LossReduction.MEAN,
+    ) -> None:
+        """
+        Args:
+            to_onehot_y (bool, optional): whether to convert `y` into the one-hot format. Defaults to False.

```

Comment:
```suggestion
            to_onehot_y: whether to convert `y` into the one-hot format. Defaults to False.

```
there's no need to include additional typing in the argument, please refer to the other modules as an example such as https://github.com/Project-MONAI/MONAI/blob/669bddf581201f994d1bcc0cb780854901605d9b/monai/losses/dice.py#L59-L66